src/nerve_tools/configure_nerve.py

```diff
@@ -82,6 +83,13 @@ def generate_subconfiguration(
 ):
 
     port = service_info.get('port')
+    # if this is a k8s pod the dict will have the pod IP and we have
+    # an hacheck sidecar in the pod that caches checks otherwise it is
+    # a marathon/puppet etc service and we use the system hacheck
+    hacheck_ip = service_info.get('hacheck_ip', '127.0.0.1')
+    # ditto for the IP of the service, in k8s this is the pod IP,
+    # otherwise we use the hosts IP
+    ip_address = service_info.get('service_ip', ip_address)
```

Comment:
I don't see this used yet.

Reply:
It is already used further down. This is just overwriting that variable if `service_ip` exists in the dict. I guess it's passed in as a variable at the moment to avoid looking up the hosts IP for every service?